insights/specs/default.py

```diff
@@ -595,7 +595,8 @@ def md_device_list(broker):
     mongod_conf = glob_file([
                             "/etc/mongod.conf",
                             "/etc/mongodb.conf",
-                            "/etc/opt/rh/rh-mongodb26/mongod.conf"
+                            "/etc/opt/rh/rh-mongodb26/mongod.conf",
```

Comment:
@Rohan21Lobo, from https://access.redhat.com/support/policy/updates/rhscl-rhel7,
 I found rh-mongodb26 is already retired, I think we can delete it. How do u think?

Reply:
I think RHUI 3 still uses mongodb26. Don't have an instance to check as of now.

Reply:
Yeap, it doesn't hurt to keep it here even `26` is retired.